profiles.yml

```diff
@@ -190,7 +190,7 @@ semantic_similarity_profiles:
   - subset: hp-fbcv
     method: semsimian
     ontology: phenio-monarch
-    threshold: 0.4
+    threshold: 0.7
```

Comment:
Why have you replaced 0.4 with 0.7 in this comparison?

Reply:
you are right, this was not necessary, good catch! Feel free to fix it; its not that important as FBCV is not tested in any of our experiments!